install/install.qmd

```diff
@@ -105,90 +106,136 @@ For more information, see the
+
+For more information, see the
+[RStan Getting Started wiki](https://github.com/stan-dev/rstan/wiki/RStan-Getting-Started){target="_blank"}
 :::::
 ::::: {.install #install-rstan-conda .hidden}
 Run `conda install -c conda-forge r-rstan`.
 
-Note: this will also install any system prerequesites.
+Note: this will also install any system prerequisites.
 :::::
 ::::: {.install #install-rstan-github-src .hidden}
-TODO 9
+Run `git clone https://github.com/stan-dev/rstan.git --recursive`
+
+For more information, see the
+[RStan Getting Started wiki](https://github.com/stan-dev/rstan/wiki/RStan-Getting-Started){target="_blank"}
 :::::
 <!-- ---------- Stan.jl ---------- -->
 ::::: {.install #install-stan.jl-julia-pkg .hidden}
-TODO 10
+To install Stan.jl e.g. in the Julia REPL: `] add Stan`,
+then use Conda.jl or conda to install CmdStan.
+
+See [Stan.jl documentation](http://stanjulia.github.io/Stan.jl/v6.0/INSTALLATION.html) for further details.
 :::::
 ::::: {.install #install-stan.jl-github-src .hidden}
-TODO 11
+Run `https://github.com/StanJulia/Stan.jl.git`,
+then follow instructions in the README file.
 :::::
 
 ::::
 :::
 
 <script src="installer.js"></script>
 
-## In-Depth Instructions
 
-### Platform
 
-#### Linux
+::: {.callout-note}
+Conda simplifies the installation process by ensuring that all required libraries and tools are compatible with each other
+and is available for Linux, Mac, and Windows platforms.
 
-- which C++
+You can either install [miniconda](https://docs.conda.io/en/latest/miniconda.html), a free, minimal installer for conda
+or you can get the full [Anaconda](https://docs.continuum.io/anaconda/) system
+which provides graphical installer wizards for [MacOS](https://www.anaconda.com/products/distribution#macos){target="_blank"}
+and [Windows](https://www.anaconda.com/products/distribution#windows){target="_blank"} users.
 
-#### MacOS
+Julia users can install [Conda.jl](https://github.com/JuliaPy/Conda.jl){target="_blank"}.
+:::
 
-- Xcode
+## Prerequisite:  C++17
 
-#### Windows
 
-- Conda
-- RTools
+Stan requires a C++17 compiler and the GNU Make utility.
 
-## Other packages and environments {#other-packages}
+* On Linux, these are bundled into the meta-package `build-essential`.
+To install, run command:\
+`sudo apt-get install build-essential`
 
-#### Google Colab
 
-#### Mathematica, Stata, etc?
+* On Mac, these are included with Xcode.
+Install Xcode from the App Store and then run command:\
+`xcode-select --install`
 
-## Troubleshooting
+* On Windows 10, we strongly recommend first installing Anaconda or miniconda and then using the conda installer to be CmdStanR or RStan,
+since the conda packages include a C++ toolchain.\
+Alternatively, R Users can get [Rtools](https://cran.r-project.org/bin/windows/Rtools/){target="_blank"} from CRAN,
+or try to install it from within CmdStanR using the `install_toolchain` method.
```

Comment:
CmdStanPy also still has a function for this, and we should link to the docs for both of them

Reply:
this function isn't listed in the CmdStanR docs, nor the CmdStanPy ones (?)

Reply:
In cmdstanpy I think it’s spelled install_cxx_toolchain

There’s also an argument to the install_cmdstan function you can pass

Reply:
added.